src/SFA.DAS.Support.Portal.Web/Views/Shared/_Layout.cshtml

```diff
@@ -35,36 +35,37 @@
     <meta name="theme-color" content="#0b0c0c"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta property="og:image" content="@Url.Content("~/assets/images/opengraph-image.png")">
-    @Html.Partial("_Analytics")
+    @Html.Partial("_GoogleAnalyticsHeadPartial")
```

Comment:
The recommendation is to place this as close to the opening `<head>` tag as possible.

Reply:
I agree, I just placed it over what was already there which was some random script that I don't think was working. Will move it up

Reply:
moved this higher